docs-src/go/generated/add-sleep-for-one-minute.md

```diff
@@ -0,0 +1,68 @@
+---
+id: add-sleep-for-one-minute
+title: Add a call to sleep
+sidebar_label: Add sleep call
+description: Add a call to sleep for one minute to the beginning of the Workflow.
+tags:
+- timer
+- sleep
+- logger
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-go/blob/port_replay_test_dacx/backgroundcheck_replay/workflows/backgroundcheck.go. -->
+
+In the following sample, we add a couple of logging statements and a Timer to the Workflow code to see how this affects the Event History.
```

Comment:
```suggestion
In the following sample, we will add a couple of logging statements and a Timer to the Workflow code to see how this affects the Event History.
```

This hasn't happened yet, so need a helper on the verb.

But then that makes it a passive voice, so should revise this sentence.

Reply:
Does changing it to "will add" make it passive?
But I think sticking with present tense is our go to -
Also, "hasn't happened yet" is an interesting perspective... could definitely argue that it has